### DISCUSSION TITLE
Willkommen zum brickmerge®-userscript Diskussionsforum!

### DISCUSSION THREAD

**Discussion by pke** [pke/brickmerge-userscript#2] [Ankündigungen]
## 👋 Willkommen!

Hier können wir uns zum brickmerge®-userscript austauschen, Shop Seiten vorschlagen, [Fehler melden](../issues) und einfach schreiben, wie toll das userscript doch ist ;)

**Comment by volmeismyusername** [comment]
![IMG_1679](https://github.com/pke/brickmerge-userscript/assets/151281857/decfd5a4-64a0-4d83-8024-1b8cb8384459)
![IMG_1682](https://github.com/pke/brickmerge-userscript/assets/151281857/4e84e5b9-1e6b-44af-8bb7-fe7c9daf0c55)
![IMG_1680](https://github.com/pke/brickmerge-userscript/assets/151281857/dbcc0232-b8ea-42a0-ab0d-78d92f2fb26e)
![IMG_1681](https://github.com/pke/brickmerge-userscript/assets/151281857/7a33c035-5033-4c87-a1a4-9b57a3fa6c98)
](url)

**Reply by volmeismyusername** [reply]
Aus dem Thread mydealz , das sind mal Screenshots von dem was „installiert“ wurde und als Beispiel  dann die Thalia  HP
ohne Einblendung

**Reply by pke** [reply]
Danke. Ich schaus mir mal an auf dem iPad

**Reply by pke** [reply]
Du musst die https://greasyfork.org/en/scripts/477807-brickmerge-prices Seite gehen. Dann den grünen Button antippen und dann den unterstrichenen Text, dass Du schon einen Userscript Manager installiert hast.
Dann öffnet er das userscript im Quelltext. Dann oben in der Adressleiste das "Userscript </>" Icon antippen und "Script installieren". Dann in dem kleinen Fenster bis nach unten scrollen und "Install" drücken.

**Reply by pke** [reply]
![image](https://github.com/pke/brickmerge-userscript/assets/164625/365cb1b8-1317-45ca-85f4-77a4363802bd)

![image](https://github.com/pke/brickmerge-userscript/assets/164625/99fc6add-c78f-4a78-a263-ff07d5a38ba2)

![image](https://github.com/pke/brickmerge-userscript/assets/164625/a8e684a6-917e-46f3-9b78-43603945b0ee)

![image](https://github.com/pke/brickmerge-userscript/assets/164625/c99ac189-c804-4aee-a2b0-0c19fb09942f)

![image](https://github.com/pke/brickmerge-userscript/assets/164625/a162a3bb-46db-41b7-b45c-e699c9e6796a)

![image](https://github.com/pke/brickmerge-userscript/assets/164625/db1a6fde-3729-4ef1-a25d-5ae53bcbab61)
